src/Command/Ingester/Native/NativeUserIngesterCommand.php

```diff
@@ -93,18 +93,19 @@ protected function configure(): void
         $this->setDescription('Responsible for native user ingesting from various sources (Local file, S3 bucket)');
 
         $this->addArgument(
-            'source',
+            'path',
             InputArgument::REQUIRED,
-            sprintf(
-                'Source type to ingest from, possible values: ["%s"]',
-                implode('", "', array_keys($this->ingesterSourceRegistry->all()))
-            )
+            'Source path to ingest from'
         );
 
         $this->addArgument(
-            'path',
-            InputArgument::REQUIRED,
-            'Source path to ingest from'
+            'source',
+            InputArgument::OPTIONAL,
+            sprintf(
```

Comment:
same here

Reply:
Same what, sorry? Which comment are you referencing to?